qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AddPersonPage.java

```diff
@@ -15,17 +15,17 @@
 public class AddPersonPage extends Page {
 
 	private static final By PERSON_NAME = By.id("personName");
-	private static final By BIRTHDATE = By.id("birthdate");
+	private static final By BIRTHDATE = By.className("hasDatepicker");
 	private static final By AGE = By.id("age");
 	private static final By GENDER_MALE = By.id("gender-M");
-	private static final By CREATE_PERSON = By.cssSelector("#createPatient input[type=\"submit\"]");
+	private static final By CREATE_PERSON_BUTTON = By.cssSelector("#createPatient input[type=\"submit\"]");
```

Comment:
Lets also try to remember, we are actually creating a patient. what do you think of   
```suggestion
	private static final By CREATE_PATIENT_BUTTON = By.cssSelector("#createPatient input[type=\"submit\"]");
```

Reply:
@sherrif10 the page is not only covering patient encounters but also when creating system users so that is why the element name would remain as CREAT_PERSON_BUTTON. The original naming covers all categories of people  because a patient is a person but with the suggested element name excludes users simply because a person in the system may not be necessarily a patient in context. cc: @mherman22